Watchman.DomainModel/ScheduleTasks/Queries/Handlers/GetScheduleTasksQueryHandler.cs

```diff
@@ -1,33 +1,27 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
+﻿using System.Linq;
 using Watchman.Cqrs;
-using Watchman.DomainModel.Tasks;
 using Watchman.Integrations.MongoDB;
 
 namespace Watchman.DomainModel.ScheduleTasks.Queries.Handlers
 {
     public class GetScheduleTasksQueryHandler : IQueryHandler<GetScheduleTasksQuery, GetScheduleTasksQueryResult>
     {
-        private readonly ISessionFactory sessionFactory;
+        private readonly ISessionFactory _sessionFactory;
 
         public GetScheduleTasksQueryHandler(ISessionFactory sessionFactory)
         {
-            this.sessionFactory = sessionFactory;
+            this._sessionFactory = sessionFactory;
         }
 
         public GetScheduleTasksQueryResult Handle(GetScheduleTasksQuery query)
         {
-            using (var session = sessionFactory.Create())
+            using var session = _sessionFactory.Create();
+            var results = session.Get<ScheduleTask>();
+            if(query.LoadOnlyActive)
             {
-                var results = session.Get<ScheduleTask>();
-                if(query.LoadOnlyActive)
-                {
-                    results = results.Where(x => !x.IsExecuted);
-                }
-                return new GetScheduleTasksQueryResult(results.ToList());
+                results = results.Where(x => !x.IsExecuted);
             }
+            return new GetScheduleTasksQueryResult(results.ToList());
```

Comment:
teraz pytanie czy chcemy robić to jako liste 
IEnumerable ma wbudowany lazy loading 
czasami to zaleta bo nie musimy pobierać całej bazy, jeśli zależy nam na 2-3 elementach, które nie wymagają przeszukiwania całości 

a czasami to wada, bo po określonych operacjach nie mamy już dostępu do tych danych 
albo musielibyśmy sprawdzać to samo kilka razy

Reply:
Ja tu w sumie nic nie zmieniałem, w sumie nie wiem na razie co lepsze, myślę, że można o tym pomyśleć kiedy już będziemy z tego korzystać